TravelPro.html

```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TravelPro</title>
+    <link rel="stylesheet" href="TravelPro.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+    <div class="navbar">
+        <div id="logo">
```

Comment:
There's heavy reliance on id for styling (id="logo", id="options", etc.). Typically, id should be used for unique elements that are targeted by JavaScript or as anchor links, not for styling purposes. Classes are more suitable for styling, especially when the styles are reused.